packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -120,6 +120,12 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
    */
   handleTabKey?: FocusZoneTabbableElements;
 
+  /**
+   * A callback method to determine if the input element should lose focus on arrow keys
+   *  @param {HTMLInputElement} inputElement The input element which is to loose focus.
+   */
```

Comment:
add default value down here.

Reply:
not sure if we could add default value to the callback method. Added @returns